src/htmx.js

```diff
@@ -2418,6 +2418,9 @@ return (function () {
             if (attributeValue) {
                 var str = attributeValue.trim();
                 var evaluateValue = evalAsDefault;
+                if (str.indexOf("unset") === 0 && str.length === 5) {
```

Comment:
Maybe I'm missing something, but why not just `str === "unset"`?

Reply:
Fair point. Will change that tomorrow.